css/style.css

```diff
@@ -263,36 +263,27 @@ input[type="checkbox"] {
 /* ----------- UI - Graph ----------- */
 
 .vis {
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
-  padding: 30px;
-}
-
-.vis > div {
-  display: flex;
-  align-items: flex-start;
-  justify-content: space-between;
-  height: 100%;
+  display: grid;
+  grid-template-rows: 75% 25%;
+  position: relative;
 }
 
-.vis > div > div {
-  display: flex;
-  flex-direction: column;
-  height: 100%;
+.vis-panel {
+  padding: 30px;
+  padding-bottom: 0;
```

Comment:
Part 3 of 3 is to remove `min-width: 780px` from `main` (circa line 202 in the CSS). You didn't edit that line in this PR; it was added at some point earlier (may have been me, honestly!).